src/app/view/IAP/hooks/useClouderyOffer.ts

```diff
@@ -29,7 +44,13 @@ export const useClouderyOffer = (): ClouderyOfferState => {
         return
       }
 
-      setPopupUrl(current => current ?? href)
+      const doAsync = async (): Promise<void> => {
+        await initConnection()
+        await getSubscriptions({ skus: SKUS })
+        setPopupUrl(current => current ?? href)
+      }
+
+      void doAsync()
```

Comment:
seems a bit risky without try catch here, it isn't clear if the imported functions will handle errors gracefully

Reply:
This has been fixed in this other PR: https://github.com/cozy/cozy-flagship-app/pull/1131